zzzz_modular_occulus/code/modules/core_implant/cruciform/rituals/base.dm

```diff
@@ -112,12 +142,163 @@ Let there be light! Makes you glow for 5 minutes at a time.
 	set_personal_cooldown(user)
 	return TRUE
 
-/datum/ritual/cruciform/base/enkindle/proc/give_glow(mob/living/carbon/human/participant)
+/datum/ritual/cruciform/occulus/enkindle/proc/give_glow(mob/living/carbon/human/participant)
 	participant.set_light(l_range = 4.5, l_power = 2.5, l_color = COLOR_YELLOW)	// Slightly better than a flare, same color as the Sword of Truth's glow
 	addtimer(CALLBACK(src, .proc/take_glow, participant), effect_time)
 	to_chat(participant, SPAN_NOTICE("A bright golden light seems to start emanating from your chest."))
 
-/datum/ritual/cruciform/base/enkindle/proc/take_glow(mob/living/carbon/human/participant)
+/datum/ritual/cruciform/occulus/enkindle/proc/take_glow(mob/living/carbon/human/participant)
 	if (participant) // check if participant still exists
 		participant.set_light(l_range = 0, l_power = 0, l_color = COLOR_YELLOW)
 		to_chat(participant, SPAN_NOTICE("The bright golden light flickers and fades from your chest."))
+
+
+/datum/ritual/cruciform/occulus/freeedagger
+	name = "Burninate"
```

Comment:
This is a crappy name

Reply:
Oh what the heck, burnination is the coolest possible thing in existance!

Reply:
It is literally a reference to a semi-retarded character in South park.